jdaviz/configs/specviz/helper.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, *args, **kwargs):
                                handler=self._redshift_listener)
 
     def load_spectrum(self, data, data_label=None, format=None, show_in_viewer=True,
-                      spectrum_viewer_reference_name=None):
+                      spectrum_viewer_reference_name=None, concat_by_file=False):
```

Comment:
I've now added this kwarg which allows users like @orifox to get their desired behavior, but only when they ask for it. By default, no concatenated spectrum is assembled or loaded.

Reply:
`load_spectrum` doesn't have any API docstring... but maybe it should (I did see that you added the new entry in the parser docstring, but not sure how easy that will be for users to find)?

Should this change (even though its an _addition_) warrant pushing back to 3.1 instead of 3.0.3?

Reply:
> load_spectrum doesn't have any API docstring

I was a bit surprised when I edited the call signature that there was no docstring to append it to. Do you think I should write one up in this PR?

> 3.1 instead of 3.0.3

The only reason I labeled earlier was because @orifox has wanted this for a long time 😅

Reply:
Ori knows how to install dev version. 😉 

I don't think we should backport either.

Reply:
I also don't see any new docstring here, as requested above. I agree that would be nice to have even if just to set `load_spectrum.__doc__ = load_data.__doc__` (see if that is even possible and if they are indeed completely equivalent, if not just type it out).